charts/load-test-environment/templates/deployment.yaml

```diff
@@ -4,14 +4,16 @@
 
 {{- $index := 0 -}}
 
-
-{{- range .Values.deployments }}
+{{- range .Values.deployments -}}
 {{- $index = add1 $index -}}
 {{- $latency := default "0" .latency -}}
-{{- $latencyVariation := default "0" .latencyVariation   -}}
-{{- $indexString := toString $index  -}}
+{{- $latencyVariation := default "0" .latencyVariation -}}
+{{- $indexString := toString $index -}}
+
+{{- $uniqueDeployName := printf "%s-lat%s-latvar%s-index%s" .name $latency $latencyVariation $indexString }}
+
+---
 
```

Comment:
I fixed it removing the newline

Reply:
It was because you put an end line chomper at the end so the next iteration of the `range` does not have a newline: https://github.com/newrelic/nri-prometheus/pull/270/files/91f213dcca0d565c92d0b46e4fdbe25682d5f9b7#diff-812ad1bdf11f045fde10edcd67619f2ba9e99b886e87734f42a5d69672d0bb99R69